### PR TITLE
Add installation page and deployment configuration guides

### PR DESCRIPTION
**Pull request by elmiko**
This change adds an installation page to the site. The new page follows a similar pattern as the blog landing page, and contains links to the official docs as well as the content from the [okd deployment configuration guides](https://github.com/elmiko/okd-deployment-configuration-guides) repository.

### CODE REVIEW

**Comment by elmiko** [comment]
this is still a draft, but i've got the basic functionality working. i would like to clean it up and perhaps talk with the docs working group about a preferred format for these docs. i also need to add a readme entry describing how to create more guides.

here is a sample of what it looks like currently:
okdio-installation-preview

File: source/guides/2021-03-24-AWS-IPI-default-deployment.html.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: AWS IPI Default Deployment
+authors:
+  - "@elmiko"
+last-updated: "2021-03-24"
+okd-version: "4.7"
+---
+
+This describes the resources used by OpenShift after perfoming an installation
```
Comment (TristanCacqueray) [inline source/guides/2021-03-24-AWS-IPI-default-deployment.html.markdown:9]: Shouldn't we refer to OKD here and in the other similar docs?
```suggestion
This describes the resources used by OKD after perfoming an installation
```

Comment (elmiko) [reply source/guides/2021-03-24-AWS-IPI-default-deployment.html.markdown:9]: good spot, i will update

**Comment by spotzz** [comment]
Do we want to have older versions of the Installation docs available as well?

**Comment by elmiko** [comment]
> Do we want to have older versions of the Installation docs available as well?

i think in the future that is a good consideration, but for the first draft of the guides we had been explicitly targeting the current release.

**Comment by elmiko** [comment]
updated
* changed "OpenShift" references to "OKD"
* added comment in README.md about adding more guides

**Comment by elmiko** [comment]
update
* remove old guides link from navbar

this should be good to merge now @dmueller2001

**Pull request merged**